sql/core/src/main/scala/org/apache/spark/sql/execution/metric/SQLMetrics.scala

```diff
@@ -163,6 +171,8 @@ object SQLMetrics {
         Utils.bytesToString
       } else if (metricsType == TIMING_METRIC) {
         Utils.msDurationToString
+      } else if (metricsType == NS_TIMING_METRIC) {
+        duration => Utils.msDurationToString(duration / 1000 / 1000)
```

Comment:
will this string lose the nanosecond precision?

Reply:
Maybe it's ok, as I test this locally with UT in SQLMetricsSuites, result below:
```
shuffle records written: 2
shuffle write time total (min, med, max): 37 ms (37 ms, 37 ms, 37 ms)
shuffle bytes written total (min, med, max): 66.0 B (66.0 B, 66.0 B, 66.0 
```
In the actual scenario the shuffle bytes written will be more larger, and keep the time to ms maybe enough, WDYT?